app.py

```
@@ -1,36 +1,79 @@
 import streamlit as st
 from pathlib import Path
 import json
+import subprocess
+import sys
+import os
+import atexit
+import signal
 # For some reason the windows version only works if this is imported here
 import pyopenms
```

Ruff (0.8.2)

`pyopenms` imported but unused

Remove unused import: `pyopenms`

(F401)